src/auth/auth.go

```diff
@@ -41,18 +41,28 @@ const (
 	// can be created for this user (during auth activation) and they cannot
 	// be removed from the set of cluster super-admins.
 	RootUser = "pach:root"
-)
 
-// ParseScope parses the string 's' to a scope (for example, parsing a command-
-// line argument.
-func ParseScope(s string) (Scope, error) {
-	for name, value := range Scope_value {
-		if strings.EqualFold(s, name) {
-			return Scope(value), nil
-		}
-	}
-	return Scope_NONE, errors.Errorf("unrecognized scope: %s", s)
-}
+	// PpsUser is a special, unrevokable cluster administrator account used by PPS
+	// to create pipeline tokens, close commits, and do other necessary PPS work.
+	// It's not possible to authenticate as ppsUser (pps reads the auth token for
+	// this user directly from etcd).
+	PpsUser = `pach:pps`
```

Comment:
Renamed the PPS user to `pach:pps` - all subjects with a `pach` prefix have cluster role bindings that can't be modified by users.

Reply:
Finally! Thank you!!